spec/features/merge_pact_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe "Merging a pact" do
+
+  let(:pact_content) { load_fixture('a_consumer-a_provider.json') }
+  let(:path) { "/pacts/provider/A%20Provider/consumer/A%20Consumer/version/1.2.3" }
+  let(:response_body_json) { JSON.parse(subject.body) }
+
+  subject do
+    patch path, pact_content, {'CONTENT_TYPE' => 'application/json' }; last_response
+  end
+
+  context "when a pact for this consumer version does not exist" do
+    it "returns a 201 Created" do
+      expect(subject.status).to be 201
+    end
+
```

Comment:
Now I think about it, I believe it's actually "can" create, rather than "should", choice left up to implementation. The number of "cans" rather than "shoulds" is what stopped it from being implemented in Webmachine in the end I think. There were so many paths it could take.